tests/utils/test_io_utils.py

```diff
@@ -91,3 +92,28 @@ def test_read_cameras_txt_nonexistent_file() -> None:
 
 
 # TODO in future PR: add round-trip test on write poses to images.txt, and load poses (poses->extrinsics->poses)
+
+
+def test_round_trip_images_txt() -> None:
+    """Loads a pose from file that was produced after applying an alignment transformation to a camera frustum. Converts
+    sample pose into extrinsics. Then converts these extrinsics back into a pose and verifies it matches
+    with the original."""
+
+    # Setup dummy pinhole camera object
+    default_intrinsics = Cal3Bundler(fx=100, k1=0, k2=0, u0=0, v0=0)
+
+    pose_from_file = np.load("gtsfm_incorrect_alignment_cameras_wTc.npy")
```

Comment:
thanks for the PR, Adi. Could you provide the pose matrix values as floats in the .py file, rather than in a numpy file? Just 16 numbers in the file you have here, right?